requirements/requirements_ansible.txt

```diff
@@ -102,21 +104,21 @@ pyparsing==2.4.0          # via packaging
 python-dateutil==2.6.1    # via adal, azure-storage, botocore
 pyvmomi==6.5
 pywinrm[kerberos]==0.3.0
-pyyaml==5.1               # via azure-cli-core, knack, openstacksdk
+pyyaml==5.1               # via ansible-tower-cli, azure-cli-core, knack, openstacksdk
 requests-credssp==1.0.2
 requests-kerberos==0.12.0  # via pywinrm
 requests-ntlm==1.1.0      # via pywinrm
 requests-oauthlib==1.2.0  # via msrest
-requests==2.21.0
+requests==2.22.0
 requestsexceptions==1.4.0  # via openstacksdk
 rsa==4.0                  # via google-auth
 s3transfer==0.1.13        # via boto3
 selectors2==2.0.1         # via ncclient
-six==1.12.0               # via azure-cli-core, bcrypt, cryptography, google-auth, isodate, keystoneauth1, knack, munch, ncclient, openstacksdk, ovirt-engine-sdk-python, packaging, pynacl, pyopenssl, python-dateutil, pyvmomi, pywinrm, stevedore
+six==1.12.0               # via ansible-tower-cli, azure-cli-core, bcrypt, cryptography, google-auth, isodate, keystoneauth1, knack, munch, ncclient, openstacksdk, ovirt-engine-sdk-python, packaging, pynacl, pyopenssl, python-dateutil, pyvmomi, pywinrm, requests-credssp, stevedore
 stevedore==1.30.1         # via keystoneauth1
 tabulate==0.8.2           # via azure-cli-core, knack
 typing==3.6.6; python_version < '3' # via msrest
-urllib3==1.24.3           # via requests
+urllib3==1.25.3           # via requests
```

Comment:
Let's avoid this upgrade, it has behavioral side effects we'll want to carefully consider, and it's the reason why I helped make the `1.24.3` backport:

https://urllib3.readthedocs.io/en/latest/user-guide.html#ssl

![image](https://user-images.githubusercontent.com/214912/59106750-2def2580-8905-11e9-9514-d8d8c48fc1ce.png)

Context:

https://github.com/urllib3/urllib3/issues/1553#issuecomment-486785584

Reply:
if this needs to be pinned for a significant amount of time then we should consider putting it in the `.in` files so we can track the reason and unblock people from running the pip-compile tool.